src/database/mod.rs

```diff
@@ -130,229 +126,103 @@ impl Database {
         Ok(Self { pool })
     }
 
-    pub async fn insert_pending_identity(
+    pub async fn insert_identity_if_not_duplicate(
```

Comment:
I'm not sure this name captures that it's a nop on a duplicate identity. Perhaps `insert_identity_or_nop_if_duplicate`?

Reply:
Changed to `insert_indentity_if_does_not_exist`. `nop` in a name sounds really bad to me.